src/app/common/resolvers/dataset-view/dataset-flows-tab.resolver.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright Kamu Data, Inc. and contributors. All rights reserved.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the LICENSE file.
+ */
+
+import { ResolveFn } from "@angular/router";
+import { datasetOverviewTabResolver } from "./dataset-overview-tab.resolver";
+import { DatasetOverviewTabData } from "src/app/dataset-view/dataset-view.interface";
+import { Observable } from "rxjs";
+
+export const datasetFlowsTabResolver: ResolveFn<Observable<DatasetOverviewTabData>> = (route, state) => {
```

Comment:
Why do we resolve Observable, and not just `DatasetOverviewTabData` ?

Reply:
Fixed, removed `Observable ` type